src/commands/build.rs

```diff
@@ -1,16 +1,84 @@
 pub async fn build(config: &fpm::Config) -> fpm::Result<()> {
     tokio::fs::create_dir_all(config.build_dir()).await?;
 
-    let (documents, config) = fpm::get_documents_with_config(config).await?;
+    let translation_snapshots = if let Some(ref original) = config.package.translation_of.as_ref() {
+        let translation_config = config.read_translation().await?;
+        let documents = config
+            .get_translation_documents()
+            .await?
+            .iter()
+            .map(|v| {
+                (
+                    v.get_id()
+                        .replace(&format!(".packages/{}/", original.name), ""),
+                    v.to_owned(),
+                )
+            })
+            .collect::<Vec<(String, fpm::File)>>();
+
+        std::env::set_current_dir(
+            &config
+                .original_directory
+                .join(".packages")
+                .join(original.name.as_str()),
```

Comment:
This is wrong. Read the docs of `.original_directory`. You have to use `.root` here.

Also you can't change directory like this. Code must be written with some assumption about what folder we are in. Pass the folder to function that need to fetch data from other folders. No set_current_dir allowed other than by central config thing which sets the current directory to the package root. And this is not dependency package root, this is the root of the package that is being built. Every other place construct the file path correctly and pass whatever is needed to do so, instead of relying on current directory.